testsuite/forge.py

```diff
@@ -701,6 +702,11 @@ def format_github_info(context: ForgeContext) -> str:
         .strip()
     )
 
+def get_testsuite_images(context: ForgeContext) -> str:
+    if context.forge_test_suite == "compat":
```

Comment:
Rather than pass test suite, why dont we just compare image tag and upgrade image tag, if they're different then print with little arrow?

I think the test suite conditionals seeping in through the code breaks the abstraction of the wrapper vs the forge cli

Reply:
test suite is pretty fundamental to how Forge works though. I've done what you suggested, but it's still not a clean abstraction since we need to print out what test suite runs (suite is part of ForgeContext still)